src/gym_electric_motor/physical_systems/electric_motors/SixPhase_PMSM.py

```diff
@@ -0,0 +1,169 @@
+        "interval": None,
+        "random_init": None,
+        "random_params": (None, None),
+    }
+
+    _model_constants = None
+
+    _initializer = None
+
+    def __init__(
+        self,
+        motor_parameter=None,
+        nominal_values=None,
+        limit_values=None,
+        motor_initializer=None,
+    ):
+        # Docstring of superclass
+        nominal_values = nominal_values or {}
+        limit_values = limit_values or {}
+        super().__init__(motor_parameter, nominal_values, limit_values, motor_initializer)
+        self._update_model()
+        self._update_limits()
+    
+    def _update_model(self):
+        """Updates the motor's model parameters with the motor parameters.
+
+        Called internally when the motor parameters are changed or the motor is initialized.
+        """
+        mp = self._motor_parameter
+        self._model_constants = np.array([
+            [-mp['r_s'], mp['l_q'], 1,          0,          0,             0, 0,         0,           0, 0,         0,         0,  0],
+            [         0,         0, 0, -mp['r_s'], -mp['l_d'], -mp['psi_PM'], 1,         0,           0, 0,         0,         0,  0],
+            [         0,         0, 0,          0,          0,             0, 0, -mp['r_s'], -mp['l_y'], 1,         0,         0,  0],
+            [         0,         0, 0,          0,          0,             0, 0,          0,          0, 0, -mp['r_s'], mp['l_x'], 1]
+
+        ])
+        self._model_constants[self.I_SD_IDX] = self._model_constants[self.I_SD_IDX] / mp["l_d"]
+        self._model_constants[self.I_SQ_IDX] = self._model_constants[self.I_SQ_IDX] / mp["l_q"]
+        self._model_constants[self.I_SX_IDX] = self._model_constants[self.I_SX_IDX] / mp["l_x"]
+        self._model_constants[self.I_SY_IDX] = self._model_constants[self.I_SY_IDX] / mp["l_y"]
+
+
+    def electrical_ode(self, state, u_dqxy, omega, *_):
+        """
+        The differential equation of the Six phase PMSM.
+
+        Args:
+            state: The current state of the motor. [i_sd, i_sq, i_sx, i_sy]
+            omega: electrical rotational speed
+            u_qdxy: The input voltages [u_sd, u_sq, u_sx, u_sy]
+
+        Returns:
+            The derivatives of the state vector d/dt([i_sd, i_sq, i_sx, i_sy])
+        """
+        return np.matmul(
+            self._model_constants,
+            np.array(
+                [
+                    state[self.I_SD_IDX],
+                    omega * state[self.I_SQ_IDX],
+                    u_dqxy[0],
+                    state[self.I_SQ_IDX],
+                    omega * state[self.I_SD_IDX],
+                    omega,
+                    u_dqxy[1],
+                    state[self.I_SX_IDX],
+                    omega * state[self.I_SY_IDX],
+                    u_dqxy[2],
+                    state[self.I_SY_IDX],
+                    omega * state[self.I_SX_IDX],
+                    u_dqxy[3]
```

Comment:
Order of these regressors is rather unsorted. Please implement the same ordering as in other motors, i.e. group the current components, the voltage components and the current * omega components. 
The same reordering must then be applied to the parameter matrix (self._model_constants).